timely_beliefs/beliefs/classes.py

```diff
@@ -337,6 +347,7 @@ def search_session(  # noqa: C901  # todo: remove after removing deprecated argu
         :param place_beliefs_in_sensor_timezone: if True (the default), belief times are converted to the timezone of the sensor
         :param place_events_in_sensor_timezone: if True (the default), event starts are converted to the timezone of the sensor
         :param custom_filter_criteria: optionally pass additional filters, such as ones that rely on subclasses
+        :param custom_join_targets: optionally pass additional join targets, to accommodate filters that rely on subclasses
```

Comment:
Is this new feature maybe more powerful than that? Could we also rely on other tables than our standard set?

Reply:
Good point.